linodecli/plugins/obj/helpers.py

```diff
@@ -4,7 +4,7 @@
 from argparse import ArgumentTypeError
 from datetime import datetime
 
-from terminaltables import SingleTable
+from rich.table import Table
 
```

Comment:
Need to add `rich.Text` import to accommodate @Dorthu suggestion in comment below
```suggestion
from rich.text import Text

```